resources/install.ps1

```diff
@@ -12,7 +12,11 @@ if (-not (Get-Command -Name spicetify -ErrorAction SilentlyContinue)) {
   Invoke-WebRequest @Parameters | Invoke-Expression
 }
 
-$spiceUserDataPath = "$env:APPDATA\spicetify"
+spicetify path userdata | Out-Null
```

Comment:
I don't really understand PowerShell at all, but is this line doing anything? Doesn't look like it's assigning any variable or anything

Reply:
#550

Reply:
It's just outputting it to console?

Reply:
calls without output

Reply:
or you are speaking about "config has been generated" message?